docs/notebooks/Common_Gotchas_in_JAX.md

```diff
@@ -219,6 +240,7 @@ For example, the update above can be written as:
 :id: PBGI-HIeCP_s
 :outputId: de13f19a-2066-4df1-d503-764c34585529
 
+jax_array = jnp.zeros((3,3), dtype=jnp.float32)
```

Comment:
Corresponding change is missing from the ipynb

Reply:
Rookie mistake...